atlp-rwanda/e-comm-team-brogrammers-fn
# #184948147 Setup deployment and automatic deploys on the base branch

#### What does this PR do?
*  This PR contains set up deployment and automatic deploys on the base branch for BrogrammersMall application.

#### Description of Task to be completed?
*  The task is to set up deployment and automatic deploys for an application on the base branch(develop), and to create staging and production environments with the appropriate URLs.

#### How should this be manually tested?
*  To test the deployment and automatic deploys, the user can navigate to the provided URLs in a browser and should be able to access the live application. To test the review app mirroring the Pull Request being raised, a developer can create a Pull Request and verify that a review app is automatically created and mirrors the changes being made in the Pull Request.

#### Any background context you want to provide?
*  Setting up deployment and automatic deploys is an important step in making an application available to other users. It ensures that the application is always up-to-date and accessible to users, and allows developers to test changes in a review app before merging them into the base branch.

#### What are the relevant pivotal tracker/Trello stories?
* [#184948147](https://www.pivotaltracker.com/n/projects/2639400/stories/184948147)

#### Screenshots (if appropriate)
* 
#### Questions:
N/A
